ouster_ros/include/ouster_ros/motion_correction.h

```diff
@@ -0,0 +1,90 @@
+#ifndef OUSTER_ROS_MOTION_CORRECTION_H_
+#define OUSTER_ROS_MOTION_CORRECTION_H_
+
+#include <kindr/minimal/quat-transformation.h>
+#include <pcl_ros/point_cloud.h>
+#include <ros/ros.h>
+#include <list>
+#include "ouster_ros/point_os1.h"
+
+namespace ouster_ros {
+namespace OS1 {
+
+typedef kindr::minimal::QuatTransformationTemplate<float> Transformation;
```

Comment:
I guess this is float to match PCL types?

Reply:
the pointcloud is floats so this saves having to convert them to doubles and back.